File: backend/poetry.py
# Generates poetry prompts


import random

@app.route('/poetry')
def generate_poetry_prompt():
    prompts = [
        "Write a poem about the beauty of nature.",
        "Compose a poem inspired by a childhood memory.",
        "Craft a sonnet expressing love and longing."
        # I will add more poetry prompts here
    ]
    return random.choice(prompts)

File: backend/poetry.js
// backend/poetry.js

const express = require('express');
const router = express.Router();


const poetryPrompts = [
  "Write a poem about the changing seasons.",
  "Create a poem that captures the feeling of a first love.",
  "Write a poem inspired by the sounds of your city.",
  "Compose a poem about the concept of time.",
  "Write a poem that explores the theme of solitude."
];

function getRandomPoetryPrompt() {
  const randomIndex = Math.floor(Math.random() * poetryPrompts.length);
  return poetryPrompts[randomIndex];
}

module.exports = getRandomPoetryPrompt;
module.exports = router;